build/images/flow-aggregator/Dockerfile

```diff
@@ -5,16 +5,17 @@ WORKDIR /antrea
 
 COPY . /antrea
 
-# Make sure the flow-aggregator binary is statically linked.
-RUN CGO_ENABLED=0 make flow-aggregator
+RUN make flow-aggregator antctl-ubuntu
 
-FROM scratch
+# Chose this base image so that a shell is available for users to exec into the container, run antctl and run tools like pprof easily
+FROM ubuntu:20.04
 
 LABEL maintainer="Antrea <projectantrea-dev@googlegroups.com>"
 LABEL description="The docker image for the flow aggregator"
 
 ENV USER root
```

Comment:
should we switch back to build arg USER? 
I remember this was something added for the SCRATCH based image.

Reply:
More explanation. 
We don't need to run root privileges to run the flow aggregator executable. 

>ARG is only available during the build of a Docker image (RUN etc), not after the image is created and containers are started from it (ENTRYPOINT, CMD). You can use ARG values to set ENV values to work around that.

Reply:
The image is built successfully without USER root in my test. Thanks, remove it.